### ISSUE TITLE
Turn off reading of alt text descriptions on web pages

### ISSUE THREAD

**Issue by Gene703122** [nvaccess/nvda#18151]
### Is your feature request related to a problem? Please describe.
At present, as far as I can tell, NVDA has no way for alt text description reading to be turned off. The user should be able to control this as on many sites no useful information is given and it is a real waste f time to hear them. 
### Describe the solution you'd like
This should be controllable by the user.
### Describe alternatives you've considered
None.

**Comment by seanbudd** [comment]
We should consider solving #7412 while working on this

**Comment by ashu-choudhury** [comment]
I'd like to work on this. Let me know if it's okay to proceed!

**Comment by seanbudd** [comment]
Hi @ashu-choudhury , since the issue has triaged label you are free to work on it
